### DISCUSSION TITLE
Get global edge indexes after getting mini batches through the Dataloader

### DISCUSSION THREAD

**Discussion by purvanshi** [pyg-team/pytorch_geometric#6717] [Q&A]
I want to map the edge indexes from every batch to the original edge indexes. Do you know if there is any internal mapping dictionary present that we can access?

**Comment by EdisonLeeeee** [comment]
There is a `e_id` attribute in each batch, which maps each edge to the original edge indexes.

**Reply by rusty1s** [reply]
(important to note that this was just recently added). For PyG <= 2.2, you can do
```python
data.e_id = torch.arange(data.num_nodes)
loader = NeighborLoader(data, ...)
